gcc-8 build error

```
../common/src/View/Autosaver.cpp
../common/src/View/Autosaver.cpp: In member function â€˜void TrenchBroom::View::Autosaver::autosave(TrenchBroom::View::MapDocumentSPtr)â€™:
../common/src/View/Autosaver.cpp:97:42: error: catching polymorphic type â€˜class FileSystemExceptionâ€™ by value [-Werror=catch-value=]
             } catch (FileSystemException e) {
                                          ^
../common/src/View/Autosaver.cpp: In member function â€˜TrenchBroom::IO::WritableDiskFileSystem TrenchBroom::View::Autosaver::createBackupFileSystem(const TrenchBroom::IO::Path&) constâ€™:
../common/src/View/Autosaver.cpp:110:42: error: catching polymorphic type â€˜class FileSystemExceptionâ€™ by value [-Werror=catch-value=]
             } catch (FileSystemException e) {
                                          ^
../common/src/View/Autosaver.cpp: In member function â€˜void TrenchBroom::View::Autosaver::thinBackups(TrenchBroom::IO::WritableDiskFileSystem&, TrenchBroom::IO::Path::List&) constâ€™:
../common/src/View/Autosaver.cpp:159:46: error: catching polymorphic type â€˜class FileSystemExceptionâ€™ by value [-Werror=catch-value=]
                 } catch (FileSystemException e) {
                                              ^
cc1plus: all warnings being treated as errors
```

--- a/common/src/View/Autosaver.cpp
+++ b/common/src/View/Autosaver.cpp
@@ -94,7 +94,7 @@ namespace TrenchBroom {
                 if (m_logger != nullptr)
                     m_logger->info("Created autosave backup at %s", backupFilePath.asString().c_str());
                 
-            } catch (const FileSystemException& e) {
+            } catch (const FileSystemException&) {
                 if (m_logger != nullptr)
                     m_logger->error("Aborting autosave");
             }

--- a/common/src/View/Autosaver.cpp
+++ b/common/src/View/Autosaver.cpp
@@ -94,7 +94,7 @@ namespace TrenchBroom {
                 if (m_logger != nullptr)
                     m_logger->info("Created autosave backup at %s", backupFilePath.asString().c_str());
                 
-            } catch (FileSystemException e) {
+            } catch (const FileSystemException& e) {
                 if (m_logger != nullptr)
                     m_logger->error("Aborting autosave");
             }
@@ -107,7 +107,7 @@ namespace TrenchBroom {
             try {
                 // ensures that the directory exists or is created if it doesn't
                 return IO::WritableDiskFileSystem(autosavePath, true);
-            } catch (FileSystemException e) {
+            } catch (const FileSystemException& e) {
                 if (m_logger != nullptr)
                     m_logger->error("Cannot create autosave directory at %s", autosavePath.asString().c_str());
                 throw e;
@@ -156,7 +156,7 @@ namespace TrenchBroom {
                     if (m_logger != nullptr)
                         m_logger->debug("Deleted autosave backup %s", filename.asString().c_str());
                     backups.erase(std::begin(backups));
-                } catch (FileSystemException e) {
+                } catch (const FileSystemException& e) {
                     if (m_logger != nullptr)
                         m_logger->error("Cannot delete autosave backup %s", filename.asString().c_str());
                     throw e;

--- a/common/src/View/MapFrame.cpp
+++ b/common/src/View/MapFrame.cpp
@@ -261,7 +261,7 @@ namespace TrenchBroom {
                     return true;
                 }
                 return saveDocumentAs();
-            } catch (FileSystemException e) {
+            } catch (const FileSystemException& e) {
                 ::wxMessageBox(e.what(), "", wxOK | wxICON_ERROR, this);
                 return false;
             } catch (...) {
@@ -283,7 +283,7 @@ namespace TrenchBroom {
                 m_document->saveDocumentAs(path);
                 logger()->info("Saved " + m_document->path().asString());
                 return true;
-            } catch (FileSystemException e) {
+            } catch (const FileSystemException& e) {
                 ::wxMessageBox(e.what(), "", wxOK | wxICON_ERROR, this);
                 return false;
             } catch (...) {
@@ -310,7 +310,7 @@ namespace TrenchBroom {
                 m_document->exportDocumentAs(format, path);
                 logger()->info("Exported " + path.asString());
                 return true;
-            } catch (FileSystemException e) {
+            } catch (const FileSystemException& e) {
                 ::wxMessageBox(e.what(), "", wxOK | wxICON_ERROR, this);
                 return false;
             } catch (...) {

